community/asset_library/submitting_to_assetlib.rst

```diff
@@ -59,6 +59,11 @@ These things are not required for your asset to be approved, but
 if you follow these recommendations, you can help make the asset
 library a better place for all users.
 
+* When creating non-project assets, it is common practice to place your files
+  inside of an **addons/AssetName/** folder. Do this to avoid having your files 
```

Comment:
```suggestion
  inside of an **addons/asset_name/** folder. Do this to avoid having your files 
```

We want to encourage lowercase file names to avoid issues with case sensitivity.